src/trackedit/internal/au3/au3trackeditclipboard.cpp

```diff
@@ -90,3 +75,68 @@ bool Au3TrackeditClipboard::isMultiSelectionCopy() const
 {
     return m_isMultiSelectionCopy;
 }
+
+std::set<int64_t> Au3TrackeditClipboard::getGroupIDs(std::vector<TrackData>& tracksData)
```

Comment:
tracksData can be marked const here

```suggestion
std::set<int64_t> Au3TrackeditClipboard::getGroupIDs(const std::vector<TrackData>& tracksData)
```